src/compiler/operations.md

```diff
@@ -55,3 +55,9 @@ T-compiler has two kinds of meetings: triage and design meetings. Triage meeting
 These filters are for checking what's happening in other teams
 
 - [List of open RFCs](https://github.com/rust-lang/rust/issues?q=is%3Aopen+is%3Aissue+label%3Aproposed-final-comment-period+label%3Adisposition-merge+sort%3Aupdated-asc) (all teams) waiting for the team to discuss or check the proposal, can anything be done to help moving them forward?
+
+## Useful tips
+
+You can utilize the [GitHub Issues Dashboard](https://github.com/issues/) to create custom filters. The filters allow you to aggregate both issues and PRs from *multiple* repositories, and allows applying [advanced filters][adv-filters]. See <https://github.blog/changelog/2025-04-02-github-issues-dashboard-updates/>.
```

Comment:
This is cool (and by the way I should start using it as well) but for clarity I would link a real example. I clicked https://github.com/issues and got 42.881.679, **all** issues on GitHub  :laughing:

Maybe [this example](https://github.com/issues?q=sort%3Aupdated-desc%20is%3Aissue%20is%3Aopen%20(repo%3Arust-lang%2Frust%20OR%20repo%3Arust-lang%2Fcargo)) could be more relevant to us: all open issues on r-l/rust and r-l/cargo? Or something else

Reply:
yay thank you!! :pray: